tests/e2e/performance/test_single_pvc_clone_perf.py

```diff
@@ -0,0 +1,175 @@
+        ),
+        pytest.param(
+            *[constants.CEPHFILESYSTEM, '100', '60GB'], marks=pytest.mark.polarion_id('2341')
+        )
+    ]
+)
+@performance
+@bugzilla('1882504')
+class TestPVCSingleClonePerformance(E2ETest):
+    """
+    Test to verify single PVC clone creation and deletion performance
+    """
+
+    @pytest.fixture(autouse=True)
+    def base_setup(self, interface_type, pvc_size, pvc_factory, pod_factory):
+        """
+        create resources for the test
+        Args:
+            interface_type(str): The type of the interface
+                (e.g. CephBlockPool, CephFileSystem)
+            pvc_size: Size of the created PVC
+            pvc_factory: A fixture to create new pvc
+            pod_factory: A fixture to create new pod
+        """
+        self.pvc_obj = pvc_factory(
+            interface=interface_type,
+            size=pvc_size,
+            status=constants.STATUS_BOUND
+        )
+        self.pod_obj = pod_factory(
+            interface=interface_type,
+            pvc=self.pvc_obj,
+            status=constants.STATUS_RUNNING
+        )
+
+    def test_clone_create_delete_performance(self, interface_type, pvc_size, file_size, teardown_factory):
+        """
+        Create single clone for an existing pvc,
+        Measure clone creation times
+        Delete the created clone
+        Measure clone deletion time
+        Note: by increasing max_num_of_clones value you increase created clones number
+        """
+        logger.info(f"Running IO on pod {self.pod_obj.name}")
+        file_name = self.pod_obj.name
+        logger.info(f"File {file_name} of {file_size} size was created during IO.")
+        self.pod_obj.run_io(
+            storage_type='fs', size=file_size, fio_filename=file_name, io_direction='wo', runtime=180, bs='1024K'
+        )
+
+        # Wait for fio to finish
+        self.pod_obj.get_fio_results()
+        logger.info(f"Io completed on pod {self.pod_obj.name}.")
+
+        # Verify presence of the file on pvc
+        file_path = pod.get_file_path(self.pod_obj, file_name)
+        logger.info(f"Actual file path on the pod is {file_path}.")
+        assert pod.check_file_existence(self.pod_obj, file_path), (
+            f"File {file_name} does not exist"
+        )
+        logger.info(f"File {file_name} exists in {self.pod_obj.name}")
+
+        max_num_of_clones = 1
+        clone_creation_time_measures = []
+        clones_list = []
+        timeout = 3600
+        sc_name = self.pvc_obj.backed_sc
+        parent_pvc = self.pvc_obj.name
+        clone_yaml = constants.CSI_RBD_PVC_CLONE_YAML
+        if interface_type == constants.CEPHFILESYSTEM:
+            clone_yaml = constants.CSI_CEPHFS_PVC_CLONE_YAML
+
+        # creating single clone ( or many one by one if max_mum_of_clones > 1)
+        logger.info(f"Start creating {max_num_of_clones} clones on {interface_type} PVC of size {pvc_size}GB.")
```

Comment:
The pod which is attached to a PVC should be "down" or the PVC should not be in USE while you attempt clone.